src/components/sections/profile/overview/Achievements.tsx

```
@@ -26,18 +26,20 @@ export default function ProfileOverviewAchievements(): ReactElement {
+            {showAll ? (
+              <button onClick={() => setShowAll(false)} className="text-brand bg-transparent pt-4 flex items-center gap-1.5 cursor-pointer">
+                <Less />
+                <span>See Less</span>
+              </button>
+            ) : (
+              <button onClick={() => setShowAll(true)} className="text-brand bg-transparent pt-4 flex items-center gap-1.5 cursor-pointer">
+                <Plus />
+                <span>See All</span>
+              </button>
+            )}
+          </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)